src/main/kotlin/net/leanix/githubagent/services/WebhookService.kt

```diff
@@ -0,0 +1,85 @@
+package net.leanix.githubagent.services
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import net.leanix.githubagent.config.GitHubEnterpriseProperties
+import net.leanix.githubagent.dto.ManifestFileAction
+import net.leanix.githubagent.dto.ManifestFileUpdateDto
+import net.leanix.githubagent.dto.PushEventPayload
+import net.leanix.githubagent.shared.MANIFEST_FILE_NAME
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class WebhookService(
+    private val webSocketService: WebSocketService,
+    private val gitHubGraphQLService: GitHubGraphQLService,
+    private val gitHubEnterpriseProperties: GitHubEnterpriseProperties,
+    private val cachingService: CachingService,
+    private val gitHubAuthenticationService: GitHubAuthenticationService
+) {
+
+    private val logger = LoggerFactory.getLogger(WebhookService::class.java)
+    private val objectMapper = jacksonObjectMapper()
+
+    fun consumeWebhookEvent(eventType: String, payload: String) {
+        when (eventType.uppercase()) {
+            "PUSH" -> handlePushEvent(payload)
+            else -> {
+                logger.info("Sending event")
```

Comment:
This log is not giving any valuable information.
It should either be `debug` or should give more relevant information for eg. type, repo name etc

Reply:
Also, if we are logging another message inside `sendMessage()` then there will be too much noise in logs.
But if not  or if we have a plan to clean them up later on, then I think we are good